caas/kubernetes/clientconfig/k8s.go

```diff
@@ -134,8 +136,7 @@ func credentialsFromConfig(config *clientcmdapi.Config) (map[string]cloud.Creden
 		var authType cloud.AuthType
 		if user.Token != "" {
 			if user.Username != "" || user.Password != "" {
-				logger.Warningf("invalid AuthInfo: '%s' has both Token and User/Pass: skipping", name)
-				continue
+				return nil, errors.NotValidf("invalid AuthInfo: %q has both Token and User/Pass: skipping", name)
```

Comment:
`NotValidf` suffixes "no valid" to the message, which will make these stuttery.
"invalid AuthInfo: "name" has both Token and User/Pass: skipping not valid"